server.js

```diff
@@ -0,0 +1,79 @@
+const express = require('express');
+const expressHandlebars = require('express-handlebars');
+require('dotenv').config();
+const app = new express();
+const router = express.Router();
+const port = process.env.PORT || 3000;
+
+app.engine('handlebars', expressHandlebars());
+app.set('view engine', 'handlebars');
+
+app.set('strict routing', true);
+
+app.use(function(req, res, next) {
+  const schema = req.headers['x-forwarded-proto'];
+  const host = req.headers.host.split(':')[0];
+
+  if (schema === 'https' || host === 'local.findthemasks.com' || host === 'localhost' ) {
+    next();
+  } else {
+    res.redirect('https://' + req.headers.host + req.url);
+  }
+});
+
+app.use((req, res, next) => {
+  res.set('Cache-Control', 'public, max-age=300');
+  next();
+});
+
+app.use(express.static('public'));
+router.use(express.static('public'));
+
+router.get(['/', '/index.html'], (req, res) => {
+  res.render('index', {
+    ogTitle: '#findthemasks',
+    ogUrl: 'https://findthemasks.com/',
+    ogDescription: 'Find where you can donate your masks or other personal protective equipment (PPE) in your local area.',
```

Comment:
why set these here instead of in the template?

Reply:
Mostly just to show how to do it for now. As soon as we get this live I'll also be integrating i18n with node and translating these for social sharing in other countries